mono/mini/debugger-agent.c

```diff
@@ -10439,8 +10440,10 @@ method_commands_internal (int command, MonoMethod *method, MonoDomain *domain, g
 
 			mono_metadata_free_mh (header);
 		}
-
 		break;
+#else
+		return ERR_NOT_IMPLEMENTED;
+#endif
```

Comment:
Why do we need these changes to the `CMD_METHOD_GET_BODY` event handling code?

Reply:
We do not have any such thing as a Il2CppExceptionClause.
Since @brianradunity sayd that CMD_METHOD_GET_BODY will not work with IL2CPP, I return not supported instead.
This is better than adding in an Il2CppMonoExceptionClause stub and then compiling fake code that might not work correctly if called.

Reply:
Thanks that makes sense.